apps/web/app/api/ai/analyze-sender-pattern/route.ts

```diff
@@ -323,6 +323,7 @@ async function getEmailAccountWithRules({
           aiProvider: true,
           aiModel: true,
           aiApiKey: true,
+          aiBaseUrl: true,
```

Comment:
you sure we need this stored at the user level?

think it's best if we just support ollama/lmstudio at the env level. or is there another need for this here?